scripts/Inelastic/IndirectReductionCommon.py

```diff
@@ -933,9 +933,8 @@ def plot_reduction(workspace_name, plot_type):
 
     can_plot_contour = mtd[workspace_name].getNumberHistograms() > 1
     if (plot_type == 'Contour' or plot_type == 'Both') and can_plot_contour:
-        from mantidplot import importMatrixWorkspace
-        plot_workspace = importMatrixWorkspace(workspace_name)
-        plot_workspace.plotGraph2D()
+        from mantidqt.plotting.functions import pcolormesh
```

Comment:
Nit: all imports should happen at the top of the file, not in-line with the code for maintainability

Reply:
See comment above....

Reply:
I missed the previous comment whilst reviewing :facepalm: